Microsoft/WebTemplateStudio
# Azure names - Zero-padded timestamp

This is required for generating consistent timestamps, i.e. yyyymmddhhmmss

The single digit component should be padded with 0.

**To test:**
1. Launch the extension in preview mode
2. Go to the Azure page
3. Add any Azure service to open up the forms
4. Select a subscription
5. Make sure tha the timestamp of the generated name follows the format above (yyyymmddhhmmss).
